packages/server/src/core/internals/utils.ts

```diff
@@ -21,7 +21,8 @@ export type Overwrite<TType, TWith> = TWith extends any
         ? { [key: string]: TWith[string] }
         : number extends keyof TWith
         ? { [key: number]: TWith[number] }
-        : object)
+        : // eslint-disable-next-line @typescript-eslint/ban-types
+          {})
```

Comment:
tbh this is unrelated to this pr, I just realized that `object` intersection uglifies types, whereas `{}` does not.